terraform/modules/elasticache-redis/outputs.tf

```diff
@@ -0,0 +1,15 @@
+output "cache_security_group_id" {
```

Comment:
Nitpick. This file is named `outputs.tf` however throughout we have been using `output.tf`.